nemo_text_processing/inverse_text_normalization/es/taggers/cardinal.py

```diff
@@ -161,18 +174,35 @@ def __init__(self):
             | (NEMO_DIGIT ** 6)
         )
         numbers_up_to_million = pynini.compose(graph, digits_up_to_million).optimize()
-        self.numbers_up_to_million = numbers_up_to_million
+        self.numbers_up_to_million = numbers_up_to_million.optimize()
+
+        if input_case == INPUT_CASED:
+            graph |= capitalized_input_graph(graph)
+            graph_digit |= capitalized_input_graph(graph_digit)
+            graph_zero |= capitalized_input_graph(graph_zero)
+            # graph_exception = capitalized_input_graph(graph_exception)
+            self.graph_no_exception |= capitalized_input_graph(self.graph_no_exception).optimize()
+            self.numbers_up_to_thousand |= capitalized_input_graph(self.numbers_up_to_thousand).optimize()
 
         # don't convert cardinals from zero to nine inclusive
         graph_exception = pynini.project(pynini.closure(NEMO_SPACE, 0, 1) + (graph_digit | graph_zero), 'input')
 
-        self.graph = (pynini.project(graph, "input") - graph_exception.arcsort()) @ graph
+        self.graph = ((pynini.project(graph, "input") - graph_exception.arcsort()) @ graph).optimize()
 
         optional_minus_graph = pynini.closure(
-            pynutil.insert("negative: ") + pynini.cross("menos", "\"-\"") + NEMO_SPACE, 0, 1
+            pynutil.insert("negative: ") + pynini.cross(ES_MINUS, "\"-\"") + NEMO_SPACE, 0, 1
         )
 
         final_graph = optional_minus_graph + pynutil.insert("integer: \"") + self.graph + pynutil.insert("\"")
 
         final_graph = self.add_tokens(final_graph)
         self.fst = final_graph.optimize()
+
+    def delete_word(self, word: str):
```

Comment:
i agree, but it exists hear for consistency with the work done in EN since both graphs will work together -- would it make more sense to directly import from the english graph instead, or still in utils?

Reply:
@tbartley94 checking in on this since merge should happen soon

Reply:
Directly import from english graph and leave an issue for it to be moved in general. We want to minimzie duplicate code.

Reply:
nm, functionality is slightly different -- we can open an issue to track adjustment but may not make sense to change EN graph now because it could delay the availability of this one